dgc/MrzEncoder.cs

```diff
@@ -3,34 +3,227 @@
 using System.Text;
 using System.Text.RegularExpressions;
 
-namespace dgc
+namespace DCC
 {
     public static class MrzEncoder
```

Comment:
I would personally have a Transliteration interface, and an MRZ/MRTD implementation. Linking to the specs (doc 9303) would be nice too. For a simple example, maybe overkill, this is a really nice module so I can see it being something that people use / look at.

Reply:
Transliteration interface would make sense if we know there are more transliterations coming along, but AFAIK there aren't - only the ICAO 9303 and only for very specific fields

Reply:
Currently, this is the only implementation and you need to reference it specifically when creating the JSON, not used by any other class, so I think it would be confusing to have this as an interface.